RIOT-OS/RIOT
# slipdev: provide stdio multiplexing over SLIP

### Contribution description
This introduces SLIPMUX like stdio multiplexing to `slipdev`. It can be used in conjunction with `sliptty` from #10477.

### Testing procedure
Test border router application with `USEMODULE += slipdev_stdio` instead of `stdio_ethos` with both `sliptty` and `tunslip`. Both still should work. Also apply the testing procedures from #10477.

Also try to use `tunslip` with just `slipdev` (without `slipdev_stdio`). It should still work as well (just without stdio).

### Issues/PRs references
Requires #10477 for interaction with Linux machine
